README.md

```diff
@@ -1,125 +1,27 @@
-# HospitalRun Frontend
```

Comment:
I think we should keep the title of the page and add a short description of what the repository is underneath the title

Reply:
+1

Reply:
✅